input/resources/au-core-server.xml

```diff
@@ -100,9 +100,11 @@
           </div>
           <blockquote>
             <div>
+
               <ol>
-                <li>TBD.</li>
-              </ol>
+                <li>A server <strong>SHALL</strong> reject any unauthorized requests by returning an HTTP 401 "Unauthorized", HTTP 403 "Forbidden", or HTTP 404 "Not Found"</li>
+                <li>For Authentication and Authorization, Systems <strong>SHOULD</strong> support any SMART App Launch and SMART Backend Service Authorization. When this is not avalable, Basic Authentication <strong>MAY</strong> be used. </li>
```

Comment:
Fixed spelling / capilalisation. 
Alerted Heath. 
It all makes sense when layed out like this - I guess the main reason for addressing it was to have something rather than TBD in except the fact that we are in change control now.